pandas/core/arrays/timedelta.py

```diff
@@ -48,9 +60,53 @@ def _sub_datelike(self, other):
         raise TypeError("cannot subtract a datelike from a {cls}"
                         .format(cls=type(self).__name__))
 
+    def _evaluate_with_timedelta_like(self, other, op):
+        if isinstance(other, ABCSeries):
+            # GH#19042
+            return NotImplemented
+
+        opstr = '__{opname}__'.format(opname=op.__name__).replace('__r', '__')
```

Comment:
we ought to have a method to do exactly this in the base ExtensionOpsMixin (but for later cleanup)